test/EFCore.PG.NodaTime.FunctionalTests/BuiltInDataTypesWithNodaTimePluginEnabledTest.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL
+{
+    public class BuiltInDataTypesWithNodaTimePluginEnabledTest : BuiltInDataTypesNpgsqlTest,
+        IClassFixture<BuiltInDataTypesWithNodaTimePluginEnabledTest.BuiltInDataTypesWithNodaTimePluginEnabledFixture>
```

Comment:
@roji I needed to re-implement `IClassFixture<...>` here so that I could avoid introducing a new abstract class for BuiltInDataTypesNpgsqlTest. (BuiltInDataTypesTestBase already closes the generic `TFixture` with BuiltInDataTypesNpgsqlFixture).

Positive: No refactoring/splitting of existing `BuiltInDataTypesNpgsqlTest` => Easy + clean
Negative: Xunit will know initialize 2 fixture classes, although only one is needed

Personally, I`d go for my quick & simple implementation. Introducing a new abstract bases class with lots of generic type constraints just makes the test-base more complicated, IMO.

Thoughts?